app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -54,18 +54,19 @@ def start
       :pod_namespace   => namespace
     ))
 
-    pod = pod_definition
-    _log.info("creating pod #{pod_full_name} to analyze docker image " \
-              "#{options[:docker_image_id]}: #{pod.to_json}")
-
+    log_pref = nil
     begin
+      log_pref = "Getting Service Account #{IMAGE_INSPECTOR_SA} for pod [#{pod_full_name}]"
```

Comment:
@simon3z why not? what pod_definition actually does is retrieve a service account.
Would you prefer to remove this and have both failures generate a 'creating pod ... failed'?